internal/ooni/ooni.go

```diff
@@ -239,6 +252,23 @@ func MaybeInitializeHome(home string) error {
 	return nil
 }
 
+// MaybeMigrateLegacyConsent takes the informed consent from a legacy version of
+// OONI Probe and migrates it to the new version
+func MaybeMigrateLegacyConsent(ooniHome string) error {
+	userHome, err := homedir.Dir()
+	if err != nil {
+		return err
+	}
+	path := filepath.Join(filepath.Join(userHome, ".ooni"), "initialized")
+	if utils.FileExists(path) {
+		_, err := os.Create(utils.InformedConsentPath(ooniHome))
```

Comment:
I should use the function to Accept the consent in here.

Reply:
Actually it's maybe not so much of a good idea since all the functions in this file take as argument the `ooniHome` so it's unexpected for this one to not behave in that way. I think leaving it as-is, since it's anyways legacy compatibility code that will eventually be removed, is safer.

Reply:
sure!